test/testlib/env.go

```diff
@@ -207,12 +206,7 @@ func loadEnvVars(t *testing.T, result *TestEnv) {
 	result.SupervisorAppName = needEnv(t, "PINNIPED_TEST_SUPERVISOR_APP_NAME")
 	result.TestWebhook.TLS = &auth1alpha1.TLSSpec{CertificateAuthorityData: needEnv(t, "PINNIPED_TEST_WEBHOOK_CA_BUNDLE")}
 
-	result.SupervisorHTTPAddress = os.Getenv("PINNIPED_TEST_SUPERVISOR_HTTP_ADDRESS")
 	result.SupervisorHTTPSIngressAddress = os.Getenv("PINNIPED_TEST_SUPERVISOR_HTTPS_INGRESS_ADDRESS")
-	require.NotEmptyf(t,
-		result.SupervisorHTTPAddress+result.SupervisorHTTPSIngressAddress,
-		"must specify either PINNIPED_TEST_SUPERVISOR_HTTP_ADDRESS or PINNIPED_TEST_SUPERVISOR_HTTPS_INGRESS_ADDRESS env var (or both) for integration tests",
```

Comment:
I'm not sure I understand this. If we're no longer using the SupervisorHTTPAddress, shouldn't we be using the SupervisorHTTPSIngressAddress instead?

Reply:
Yes @margocrawf, that is indeed the essence of the change. We still use `SupervisorHTTPSIngressAddress`, and it is still optional. It is only actually used by `TestSupervisorOIDCDiscovery_Disruptive`, which will skip that part of the test when it is unset. In practice, our CI only currently sets it for GKE and TKG clusters, which is where we create an `Ingress` for incoming traffic.